langchain4j-couchbase/src/test/java/dev/langchain4j/store/embedding/couchbase/CouchbaseEmbeddingStoreCloudIT.java

```diff
@@ -0,0 +1,52 @@
+package dev.langchain4j.store.embedding.couchbase;
+
+import dev.langchain4j.data.segment.TextSegment;
+import dev.langchain4j.model.embedding.AllMiniLmL6V2QuantizedEmbeddingModel;
+import dev.langchain4j.model.embedding.EmbeddingModel;
+import dev.langchain4j.store.embedding.EmbeddingStore;
+import dev.langchain4j.store.embedding.EmbeddingStoreIT;
+import dev.langchain4j.store.embedding.EmbeddingStoreWithFilteringIT;
+import dev.langchain4j.store.embedding.couchbase.CouchbaseEmbeddingStore;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;
+
+import static dev.langchain4j.internal.Utils.randomUUID;
+
+@Disabled
+@EnabledIfEnvironmentVariable(named = "CB_CLUSTER", matches = ".+")
```

Comment:
`CB_CLUSTER` is a bit ambiguous, since LC4J contains a lot of other service implementations, so my proposal would be: `COUCHBASE_CLUSTER_URL`. The rest of env variables shall use the full service name as well.